app/_locales/ja/messages.json

```diff
@@ -1,478 +1,2102 @@
+    "message": "再試行"
+  },
   "typePassword": {
-    "message": "パスワードの入力"
+    "message": "MetaMask パスワードを入力"
+  },
+  "unapproved": {
+    "message": "未承認"
+  },
+  "units": {
+    "message": "単位"
   },
   "unknown": {
-    "message": "不明"
+    "message": "不明です"
+  },
+  "unknownCameraError": {
+    "message": "カメラにアクセスしているときにエラーが発生しました。もう一度実行してください..."
+  },
+  "unknownCameraErrorTitle": {
+    "message": "申し訳ありません。問題が発生しました..."
   },
   "unknownNetwork": {
-    "message": "不明なプライベートネットワーク"
+    "message": "不明なプライベート ネットワーク"
+  },
+  "unknownQrCode": {
+    "message": "エラー:QR コードを識別できませんでした"
+  },
+  "unlimited": {
+    "message": "無制限"
   },
   "unlock": {
-    "message": "ログイン"
+    "message": "ロック解除"
+  },
+  "unlockMessage": {
+    "message": "分散型 Web が待機しています"
+  },
+  "updatedWithDate": {
+    "message": "$1 に更新されました"
+  },
+  "urlErrorMsg": {
+    "message": "URL には適切な HTTP/HTTPS プレフィックスが必要です。"
+  },
+  "urlExistsErrorMsg": {
+    "message": "URL はネットワークの既存のリストに既に存在します"
+  },
+  "usePhishingDetection": {
+    "message": "フィッシング検出を使用"
+  },
+  "usePhishingDetectionDescription": {
+    "message": "イーサリアムユーザーを対象としたドメインのフィッシングに対して警告を表示します"
   },
   "usedByClients": {
-    "message": "様々なクライアントによって使用されています。"
+    "message": "さまざまな異なるクライアントによって使用されています"
+  },
+  "userName": {
+    "message": "ユーザー名"
+  },
+  "verifyThisTokenOn": {
+    "message": "このトークンを $1 で検証",
+    "description": "Points the user to etherscan as a place they can verify information about a token. $1 is replaced with the translation for \"etherscan\""
   },
   "viewAccount": {
-    "message": "アカウントを見る"
+    "message": "アカウントを表示"
+  },
+  "viewContact": {
+    "message": "連絡先を表示"
+  },
+  "viewOnCustomBlockExplorer": {
+    "message": "$1 に表示"
   },
   "viewOnEtherscan": {
-    "message": "Etherscan で見る"
+    "message": "Etherscan で表示"
+  },
+  "viewinExplorer": {
+    "message": "Explorer で表示"
+  },
+  "visitWebSite": {
+    "message": "当社の Web サイトにアクセス"
+  },
+  "walletConnectionGuide": {
+    "message": "当社のハードウェア ウォレット接続ガイド"
   },
   "walletSeed": {
-    "message": "ウォレットのパスフレーズ"
+    "message": "シード フレーズ"
   },
   "welcome": {
-    "message": "MetaMask ベータ版へようこそ!"
+    "message": "MetaMask にようこそ"
   },
   "welcomeBack": {
-    "message": "おかえりなさい!"
+    "message": "ようこそ"
+  },
+  "whatsThis": {
+    "message": "これは何ですか?"
+  },
+  "writePhrase": {
+    "message": "このフレーズを紙に書いて安全な場所に保管してください。さらなるセキュリティが必要な場合は、複数の紙に書いて、それぞれを 2 ～ 3 の異なる場所に保管します。"
+  },
+  "xOfY": {
+    "message": "$2 中の $1",
+    "description": "$1 and $2 are intended to be two numbers, where $2 is a total, and $1 is a count towards that total"
+  },
+  "yesLetsTry": {
+    "message": "試行する"
+  },
+  "youNeedToAllowCameraAccess": {
+    "message": "この機能を使用するには、カメラへのアクセスを許可する必要があります。"
   },
   "youSign": {
-    "message": "署名しています。"
+    "message": "著名しています"
+  },
+  "yourPrivateSeedPhrase": {
+    "message": "秘密のシード フレーズ"
+  },
+  "zeroGasPriceOnSpeedUpError": {
+    "message": "スピードアップ時にゼロのガス価格"
   }
 }
```

Comment:
Thank you for waiting.
This is 100% coverage file.It is updated for #10300.
please  replace messages.js file.

https://github.com/nyatla/metamask_PR10265/blob/master/message_translation_review20200126/messages.json

Reply:
@Gudahtt @nyatla's version has more differences than appear in the above suggested change. we should dump that message.json on top of this and review the diff again